### DISCUSSION TITLE
ConnectorX 0.2.1 Release

### DISCUSSION THREAD

**Discussion by wangxiaoying** [sfu-db/connector-x#131] [Announcement]
# ConnectorX 0.2.1 is now released!

## Features
* Adding Microsoft SQL Server as data source #105 
* Adding Oracle as data source #112 
* Support integer and float array type for Postgres #117
* Adding more types in mysql [[doc]](https://github.com/sfu-db/connector-x/blob/main/Types.md#mysql-clickhouse) #110  
* Add support for Arrow2 #116 

## Bug Fix
* Fix MySQL NULL value error #111 
* Support posix regex by updating sqlparse crate #120 
* Upgrade r2d2_postgres crate for Teiid #107

**Comment by quambene** [comment]
Just saw that v0.2.1 is not published on crates.io yet. Crates.io is handy to check the available features (https://crates.io/crates/connectorx/versions). That's why I missed that arrow2 is already supported.

**Reply by wangxiaoying** [reply]
Hi @quambene , thanks for the reminder! The newest alpha version has been published to crates.io: https://crates.io/crates/connectorx/0.2.2-alpha.3

**Reply by quambene** [reply]
Great, thanks!
